Docsite: replace Latin phrases to English (#71588)
ansible/ansible: docs/docsite/rst/dev_guide/developing_modules_documenting.rst

:description:

  * A detailed description (generally two or more sentences).
  * Must be written in full sentences, in other words, with capital letters and periods/full stops.
  * Shouldn't mention the module name.
  * Make use of multiple entries rather than using one long paragraph.
  * Don't quote complete values unless it is required by YAML.

:version_added:

  * The version of Ansible when the module was added.

>   * This is a string, and not a float, in other words, ``version_added: '2.1'``

Reviewer: ```suggestion
  * This is a string, and not a float, for example, ``version_added: '2.1'``
```

  :version_added:

    * Only needed if this option was extended after initial Ansible release, in other words, this is greater than the top level `version_added` field.

>     * This is a string, and not a float, in other words, ``version_added: '2.3'``.

Reviewer: ```suggestion
    * This is a string, and not a float, for example, ``version_added: '2.3'``.
```

  :sample:
    One or more examples.
  :version_added:
    Only needed if this return was extended after initial Ansible release, in other words, this is greater than the top level `version_added` field.

>     This is a string, and not a float, in other words, ``version_added: '2.3'``.

Reviewer: ```suggestion
    This is a string, and not a float, for example, ``version_added: '2.3'``.
```
